packages/mcp-solana/scripts/check-balances.ts

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env tsx
+
+import { Connection, PublicKey, LAMPORTS_PER_SOL } from "@solana/web3.js";
+import fs from "fs";
+import path from "path";
+import { Keypair } from "@solana/web3.js";
+import { config, NETWORKS } from "../src/config.js";
+
+async function getUSDCBalance(
+  connection: Connection,
+  walletAddress: PublicKey,
+  network: "devnet" | "mainnet",
```

Comment:
Nit, any reason not to use the type you defined in config for the network (since you're importing it and using it anyway), rather than repeating yourself?